core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java

```diff
@@ -72,4 +53,22 @@ public MetaDataStoringProject getProject() {
     public void execute() throws AntennaException {
         antennaCLIFrontend.execute();
     }
+
+    private WrappedGradleProject getWrappedGradleProject(Project gradleProject, Path propertiesFilePath) {
+        if (!this.pomFilePath.exists()) {
+            throw new IllegalArgumentException("Cannot find " + this.pomFilePath.toString());
+        }
+
+        if (propertiesFilePath != null) {
+            this.propertiesFilePath = propertiesFilePath.toFile();
```

Comment:
I would suggest to remove the unnecessary private field `propertiesFilePath` with e.g.

```diff
diff --git a/core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java b/core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java
index ca6a4f2a..44895897 100644
--- a/core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java
+++ b/core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/AntennaImpl.java
@@ -16,14 +16,13 @@ import org.eclipse.sw360.antenna.frontend.stub.cli.AbstractAntennaCLIFrontend;
 import org.gradle.api.Project;
 
 import java.io.File;
+import java.nio.file.Files;
 import java.nio.file.Path;
 
 public class AntennaImpl {
     private final AbstractAntennaCLIFrontend antennaCLIFrontend;
     private final File pomFilePath;
 
-    private File propertiesFilePath;
-
     public AntennaImpl(String pluginDescendantArtifactIdName, Path pomFilePath, Project gradleProject) {
         this(pluginDescendantArtifactIdName, pomFilePath, gradleProject, null);
     }
@@ -60,14 +59,12 @@ public class AntennaImpl {
         }
 
         if (propertiesFilePath != null) {
-            this.propertiesFilePath = propertiesFilePath.toFile();
-            if(!this.propertiesFilePath.exists()) {
-                throw new IllegalArgumentException("Cannot find " + this.propertiesFilePath.toString());
+            if(!Files.exists(propertiesFilePath)) {
+                throw new IllegalArgumentException("Cannot find " + propertiesFilePath.toString());
             } else {
-                return new WrappedGradleProject(gradleProject, pomFilePath, this.propertiesFilePath);
+                return new WrappedGradleProject(gradleProject, pomFilePath, propertiesFilePath.toFile());
             }
         } else {
-            this.propertiesFilePath = null;
             return new WrappedGradleProject(gradleProject, pomFilePath);
         }
     }
```